docs/deployment.md

```diff
@@ -0,0 +1,31 @@
+
+
+
+
+![image](images/ci_cd_github_big_picture_public.drawio.png)
+
+
+# Helm Chart
+
+![image](images/external-helm-chart.drawio.png)
+
+There is a sample chart. https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample . It consists of subcharts. Each subchart is a different module for example frontend, backend, eai.  Each module was created with `helm create` and some adaptions. The Chart.yml includes the external dependency of the apigateway. The main config is done in the values.yml of the parent chart. In the section refarch-gateway, frontend, backend you can set the config of the subchart. 
+
+
+
+
+# Deployment
+
+At the end of a sprint a developer builds a release in the source code project. It needs to be done for each folder frontend/backend/eai separately. Then the devops engineer needs to update the chart-version and app-version of the helm chart in the Chart.yaml.
+
+Internal the quay syncs the docker images with the external github image registry. The github api has a pull limit so we need to cache the images internally.
```

Comment:
You don't explain quay. I would prefer to keep this general and not use the tool name, but e.g. "internal Image repository". Also I would start with the 2nd sentence (why we are synching and the, how the synch ist done)